.changeset/violet-geckos-lick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/fiori-elements-writer': minor
+---
+
+removes trusted host validation
```

Comment:
Could you change this changelog comment to "removes unnecessary trusted host validation". In addition, I think it is more of a patch then a minor change because you are not adding or changing functionality, you are removing a faulty limitation

Reply:
Yes, done with https://github.com/SAP/open-ux-tools/pull/810/commits/e3c04dd73ac86b21e2316a925c5db9e8eaaa0b56